src/main/java/at/hannibal2/skyhanni/data/HotmData.kt

```diff
@@ -0,0 +1,474 @@
+        20,
+        { currentLevel -> (currentLevel + 1.0).pow(4) },
+        { level -> mapOf(HotmReward.UNKNOWN to 0.75 * level) }),
+    EAGER_ADVENTURER("Eager Adventurer",
+        100,
+        { currentLevel -> floor((currentLevel + 1.0).pow(2.3)) },
+        { level -> mapOf(HotmReward.MINING_SPEED to 2.0 * level) }),
+
+    DEAD_MANS_CHEST("Dead Man's Chest",
+        50,
+        { currentLevel -> floor((currentLevel + 1.0).pow(3.2)) },
+        { level -> mapOf(HotmReward.UNKNOWN to 1.0 * level) }),
+
+    GIFTS_FROM_THE_DEPARTED("Gifts from the Departed",
+        100,
+        { currentLevel -> floor((currentLevel + 1.0).pow(2.45)) },
+        { level -> mapOf(HotmReward.UNKNOWN to 0.2 * level) }),
+
+    EXCAVATOR(
+        "Excavator",
+        50,
+        { currentLevel -> (currentLevel + 1.0).pow(3) },
+        { level -> mapOf(HotmReward.UNKNOWN to 0.5 * level) }),
+    RAGS_TO_RICHES("Rags to Riches",
+        50,
+        { currentLevel -> floor((currentLevel + 1.0).pow(3.05)) },
+        { level -> mapOf(HotmReward.MINING_FORTUNE to 2.0 * level) }),
+
+    KEEN_EYE("Keen Eye", 1, { null }, { emptyMap() }), MINESHAFT_MAYHEM(
+        "Mineshaft Mayhem",
+        1,
+        { null },
+        { emptyMap() }),
+    FROZEN_SOLID(
+        "Frozen Solid",
+        1,
+        { null },
+        { emptyMap() }),
+    GEMSTONE_INFUSION("Gemstone Infusion", 1, { null }, { emptyMap() }), HAZARDOUS_MINER(
+        "Hazardous Miner",
+        1,
+        { null },
+        { emptyMap() }), ;
+
+    private val guiNamePattern by repoGroup.pattern("perk.name.${name.lowercase().replace("_", "")}", "§.$guiName")
+
+    var activeLevel: Int
+        get() = storage?.perks?.get(this.name)?.level ?: 0
+        private set(value) {
+            storage?.perks?.computeIfAbsent(this.name) { HotmTree.HotmPerk() }?.level = value
+        }
+
+    var enabled: Boolean
+        get() = storage?.perks?.get(this.name)?.enabled ?: false
+        private set(value) {
+            storage?.perks?.computeIfAbsent(this.name) { HotmTree.HotmPerk() }?.enabled = value
+        }
+
+    var isUnlocked: Boolean
+        get() = storage?.perks?.get(this.name)?.isUnlocked ?: false
+        private set(value) {
+            storage?.perks?.computeIfAbsent(this.name) { HotmTree.HotmPerk() }?.isUnlocked = value
+        }
+
+    var slot: Slot? = null
+        private set
+
+    fun getLevelUpCost() = costFun(activeLevel)
+
+    fun getReward() = rewardFun(activeLevel)
+
+    companion object {
+
+        val storage get() = ProfileStorageData.profileSpecific?.mining?.hotmTree
+
+        val abilities =
+            listOf(PICKOBULUS, MINING_SPEED_BOOST, VEIN_SEEKER, MANIAC_MINER, HAZARDOUS_MINER, GEMSTONE_INFUSION)
+
+        private val inventoryPattern by repoGroup.pattern(
+            "inventory", "Heart of the Mountain"
+        )
+
+        private val levelPattern by repoGroup.pattern(
+            "perk.level", "§7Level (?<level>\\d+).*"
```

Comment:
currently doesnt work if user is holding a drill with blue cheese omelete
```suggestion
            "perk.level", "§(?<color>.)Level (?<level>\\d+).*"
```